packages/@dcl/ecs/src/systems/crdt/index.ts

```diff
@@ -1,17 +1,28 @@
 import { crdtProtocol, Message as CrdtMessage } from '@dcl/crdt'
 
-import type { IEngine } from '../../engine'
+import type { ComponentDefinition, IEngine } from '../../engine'
 import { Entity } from '../../engine/entity'
 import { createByteBuffer } from '../../serialization/ByteBuffer'
 import { ComponentOperation as Message } from '../../serialization/crdt/componentOperation'
 import WireMessage from '../../serialization/wireMessage'
 import { ReceiveMessage, TransportMessage, Transport } from './types'
 
+/**
+ * @public
+ */
+export type OnChangeFunction = (
+  entity: Entity,
+  component: ComponentDefinition<any>,
+  componentId: number,
```

Comment:
we can avoid the componentId prop because itt's inside the component.
`component._id`